scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -78,9 +81,35 @@ def open_spider(self, spider: Spider) -> None:
         if hasattr(spider, "download_warnsize"):
             self._warn_size = spider.download_warnsize
 
+    @staticmethod
+    def _get_unsupported_suggestion(encoding: bytes) -> str:
+        if encoding == b"br":
+            return " You need to install brotli or brotlicffi to use it."
+        if encoding == b"zstd":
+            return " You need to install zstandard to use it."
+        return ""
+
     def process_request(
         self, request: Request, spider: Spider
     ) -> Request | Response | None:
+
+        user_header = request.headers.get("Accept-Encoding")
+        if user_header:
```

Comment:
```suggestion
        if user_header:
            user_header = user_header.lower()
```

While I found no resource to confirm that the value is meant to be interpreted as case-insensitive ([this](https://github.com/jquery/codeorigin.jquery.com/issues/43) was the closest), I suspect it should be.